README.md

```diff
@@ -126,6 +127,54 @@ client = Redis.new(semian: {
 })
 ```
 
+#### Net::HTTP
+For the `Net::HTTP` specific Semian adapter, since many external libraries may create
+HTTP connections on the user's behalf, the parameters are instead provided
+by calling specific functions in `Semian::NetHTTP`, perhaps in an initialization file.
+
+To give Semian parameters, assign a `proc` to `Semian::NetHTTP.semian_configuration`
+that takes two parameters, `host` and `port like `127.0.0.1` and `80` or `github_com` and `80`,
+and returns two values, a `semian_identifier` and a `Hash` with the keys as follows.
+
+```ruby
+SEMIAN_OPTIONS = { tickets: 1,
+                   success_threshold: 1,
+                   error_threshold: 3,
+                   error_timeout: 10 }
+Semian::NetHTTP.semian_configuration = proc do |host, port|
+  semian_identifier = "nethttp_#{host}_#{port}"
```

Comment:
I don't understand why identifier changed from `http_` to `nethttp_`, that leaks an abstraction of an internal Ruby library in the name, which doesn't make sense to me. I'm completely OK with not having an option to switch, but simply documenting this as a simple case instead of adding it as a default fallback, so :+1: to this except the default naming.